utilities/exceptions/invalidInput.m

```diff
@@ -0,0 +1,10 @@
+classdef invalidInput < MException
```

Comment:
What the difference btw invalid type and invalid input?

Reply:
As I was developing the exceptions Invalid input split into invalid type and invalid number of inputs. I'll remove this one and we can always readd it later if needs be.